mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1474,20 +1478,26 @@ def _test_first_row(self, eval_df):
                     name = f"{metric.name}/{metric.version}" if metric.version else metric.name
                     self.metrics_values.update({name: metric_value})
             except Exception as e:
+                stacktrace_str = traceback.format_exc()
```

Comment:
Adding stack trace since it's hard to understand the `eval_fn` error without stack trace.